spec/features/issue_pages/textile_editor_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+describe 'issue form', js: true do
+  before do
+    login_to_project_as_user
+    visit new_project_issue_path(current_project)
+  end
+
+  let(:submit_form) { click_button 'Create Issue' }
+
+  describe 'clicking the \'help\' button' do
+    before { find('form .btn-help').click }
+
+    it 'displays Textile help' do
+      expect(page).to have_selector 'h5', text: /Text styles/i
```

Comment:
Can we check for the visibility of the container instead of the text content? I feel like checking content seems brittle for the spec